tests/unit_tests/cpu/navigation/intersection/intersection_kernel.cpp

```diff
@@ -104,7 +104,7 @@ GTEST_TEST(detray_intersection, intersection_kernel_ray) {
     mask_container_t mask_store(host_mr);
 
     const rectangle_t rect{0u, 10.f, 10.f};
-    const trapezoid_t trap{0u, 10.f, 20.f, 30.f};
+    const trapezoid_t trap{0u, 10.f, 20.f, 30.f, 1.f / 60.f};
```

Comment:
Are these changes related to this PR?

Reply:
I tried to fix a maybe-uninitialized warning in the mask constructor and found that some initializers were actually missing...